polkadot/xcm/xcm-simulator/fuzzer/README.md

```diff
@@ -6,35 +6,31 @@ underflows.
 ## Install dependencies
 
 ```
-cargo install honggfuzz
+cargo install --force ziggy cargo-afl honggfuzz grcov
 ```
 
 ## Run the fuzzer
 
 In this directory, run this command:
 
 ```
-cargo hfuzz run xcm-fuzzer
+cargo ziggy fuzz
```

Comment:
It seems to be running on only one thread when i try this. Is it expected?  
Hongfuzz can use all threads.